src/Tizen.Multimedia/AudioManager/AudioDucking.cs

```diff
@@ -0,0 +1,178 @@
+        /// <exception cref="ObjectDisposedException">The ducking has already been disposed of.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsDucked
+        {
+            get
+            {
+                if (_disposed)
+                {
+                    throw new ObjectDisposedException(nameof(AudioDucking));
+                }
+
+                Interop.AudioDucking.IsDucked(Handle, out bool isDucked).
+                    ThrowIfError("Failed to get the running state of the device");
+
+                return isDucked;
+            }
+        }
+
+        /// <summary>
+        /// Activate audio ducking
+        /// </summary>
+        /// <param name="duration">The duration(milisecond) for ducking.</param>
+        /// <param name="ratio">The volume ratio after ducked.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     <paramref name="duration"/> is less than 0 or greater than 3000.<br/>
+        ///     -or-<br/>
+        ///     <paramref name="ratio"/> is less than 0.0 or greater than or equal to 1.0.<br/>
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        ///     Operation failed; internal error.<br/>
+        ///     -or-<br/>
+        ///     The target stream is already ducked.
+        /// </exception>
+        /// <exception cref="UnauthorizedAccessException">The caller does not have required privilege to set volume.</exception>
+        /// <exception cref="ObjectDisposedException">The ducking has already been disposed of.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public void Activate(uint duration, double ratio)
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException(nameof(AudioDucking));
+            }
+
+            if (duration < 0 || duration > 3000)
+            {
+                throw new ArgumentOutOfRangeException(nameof(duration), duration, "Valid range : 0 <= duration <= 3000");
+            }
+
+            if (ratio < 0.0 || ratio >= 1.0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(ratio), ratio, "Valid range : 0 <= ratio < 1.0");
+            }
+
+            Interop.AudioDucking.Activate(Handle, duration, ratio).
+                ThrowIfError("Failed to activate ducking");
+        }
+
+        /// <summary>
+        /// Deactivate audio ducking
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        ///     Operation failed; internal error.<br/>
+        ///     -or-<br/>
+        ///     The target stream is already unducked.
+        /// </exception>
+        /// <exception cref="UnauthorizedAccessException">The caller does not have required privilege to set volume.</exception>
+        /// <exception cref="ObjectDisposedException">The ducking has already been disposed of.</exception>
```

Comment:
I think it is better to use 
"The \<see cref="AudioDucking"/>'" instead of "The ducking".

Reply:
Done.